### DISCUSSION TITLE
Extra Blank Page Rendering Issue.

### DISCUSSION THREAD

**Discussion by aunjum** [diegomura/react-pdf#2222] [Q&A]
React-Pdf renders 'Extra Blank Page' IDK why! I am looking for a solution. Can anyone help me out?

![image](https://user-images.githubusercontent.com/77085509/222351340-4c153fcc-0845-4550-bf46-171d04340636.png)

Here is my code snippet:

`https://pastebin.pl/view/ded42ce9`

**Comment by jeetiss** [comment]
i can't reproduce the 'Extra Blank Page'

https://react-pdf-repl.vercel.app/?cp_code=JYWwDg9gTgLgBAbwFBzgSRAQwOYFMA0KcAKrgB4yGoAKOBRAasLgO5VwAiEAxgK4i4AdpSIAxCMPYBlGAE8ANrikALXLhEBfOADMoEEHABEAASi5M3GAFowAE20B6M4Nu4zUQwG4kSccIB0ZtjAAM4wbgAUyKjamCDA8rIAXEYAchAwEHBSmIIhcABS1IbsIVDcKYbKMDBgIUkODty2gv4AViGu8sAAblD-guoOgmAgwxkQViG5IVZtmGC5uCG4xgCM_gAMWw7aEjAhDumZOXlFVgAywNjV_hnaJUgaAJTeSNwSYXBhCstwALzZOSKFRqGD-bhmTDhKJERZ4FLRVA6faiOIJZJpCbZGaFYrsVCLWy2YCCbAAIQymRAlQAbJswGRHsi4JhuthBGhwiB6kZuEJwh4CXAiSSycQIGA6QymewNOwYMAYIpEUQYvspMAAF64SoAJhlzOR4QoAEF2YJKoptDAjagsFBgpajGtDcKWMBbDBlJUAJybACkRvlRGwUE9AGF9phSW5VSyHU7KgAORl21kWrm4HmVfnCNzpj1en1GACsgfTqmu1UqAGYK8KAEbQVxQfWM74Qbq2OCN-QWADWwfYYc9WZAa3jyNFpOwlVdabl7FAdE1OqnqCL3v1mzdargVZuMB3e9QIdQo9s471G5FmGJs8qpcXRHPcE9t72wjXupdtPTiakvOp5wCGLxvOQkCwHArixLw8jwBEzwAgAfHAEREAAPFwfACMIKH7phtB4N82q4P8himgALIY3zAuRCA_IoIT-PCuAaARLKoJhTCsHRvz_Ix9EsYqyrsZxXHcaQFD7pJzitoJUR3ngqT8I2bigch_xoQABoAv_GAIexgAUrgAigASoAzQyAD8MgCTDIAWP-AFsMVmAB0McAACQIGxqkgOpUAaDpGiyVx2jAGQuC2EFqAOBJLKYQ4vEsDFyI8cwLD8YoglMcs_iXlGwgxoMflJbFCXpQxWUsZe45rBxkXJRgdB1SyWWZcJ_grngP6BZJkllNwFHVLU9SNFClg2PYdyOg4HXLA4ACOvDAG0GS4BsbRgNghhNcibIwBRc1QHAHyuO-WB4FtPXItFdVxQlxXJaVLVCb8lXhle3J6rVl1SeQ8BPRV7W2Bxay0gA7KDep6smIOg5stZ6msVFrMmVFUZscXSTA93cfFqX3bdeNBSlfH_W1uXRrGRU3Y99GtS9OVvdVX3fZhDV4NtqCk_TM1dRz3zlANNR1A0TjmGNdjaJN2DTWds0LUtK1rRtF3fagu37Ydx24KddAq5d12XQTrDY3AxNpVzzEM2OH3M4bmNlXTluesDYMQ1DMNwwjSMo2jGO_SbRuJUTuPG0TNMCc9lvk_llO21xZsO5H2VVdyNUm9xbO4HzFvZTzZHdar_P9VUQvDaLFjWBLUsy3Qhzy8t4RK5tfPq4YB1HRAJ0zXrPUGz1gcB-HGVJ691vZp96em_bOcsc7KEw270Ng57iPI6j6MOJjAch0HkkD4RDjEbgElxTh_ACgRrxIEAA&modules=true

**Comment by Chino70** [comment]
I also always appear first white page only in FireFox, had no problems in Chrome, please can someone help me?

**Comment by tmoran-stenoa** [comment]
I've had the same issue with my code. I solved it by replacing `flex: 1` with `height: 100%` on my main component.
